fun.js

```diff
@@ -1,47 +1,144 @@
-let seconds = 00;
-let minutes = 00;
-let millsecs = 00;
-let bebop;
-
-
+'use strict';
 
-const heyy = document.getElementById('timer');
+let seconds = 0;
+let minutes = 0;
+let millsecs = 0;
+let bebop;
 
 function runOrStop() {
- if (bebop === false) {
-   document.getElementById('timer').innerHTML = minutes+":"+seconds + ":" + millsecs;
-} else if(bebop === true){
-  start();
+  if (bebop === false) {
+    stop();
+  } else if (bebop === true) {
+    start();
   }
 }
 
 setInterval(runOrStop, 10);
 
-
 function start() {
-  document.getElementById('timer').innerHTML = minutes+ ":" +  seconds + ":" + millsecs;
+  changeStopwatch();
   millsecs++;
   bebop = true;
-if (millsecs == 100) {
+  if (millsecs == 100) {
     seconds++;
     millsecs = 0;
   }
-if (seconds == 60) {
-      minutes++;
-      seconds = 0;
-    }
-
+  if (seconds == 60) {
+    minutes++;
+    seconds = 0;
+  }
 }
 
-
-
 function stop() {
   bebop = false;
+  changeStopwatch();
 }
 
 function reset() {
-  seconds = 00;
-  millsecs = 00;
-  minutes = 00;
+  seconds = 0;
+  millsecs = 0;
+  minutes = 0;
   bebop = false;
 }
+
+function changeStopwatch() {
+  let minuteStr;
+  let secondStr;
+  let millsecStr;
+  if (millsecs < 10) {
+    millsecStr = `:0${millsecs}`;
+  } else {
+    millsecStr = `:${millsecs}`;
+  }
+  if (seconds < 10) {
+    secondStr = `:0${seconds}`;
+  } else {
+    secondStr = `:${seconds}`;
+  }
+  if (minutes < 10) {
+    minuteStr = `0${minutes}`;
+  } else {
+    minuteStr = `${minutes}`;
+  }
+  document.getElementById('stopwatch').innerHTML = minuteStr + secondStr + millsecStr
+}
+
+let timerMillsec;
+let timerSec;
+let timeMins;
+let timeInput;
+let timerOnOrOff;
+
+function timerGoOrNo() {
+  if (timerOnOrOff === false) {
+    timerStop();
+  } else if (timerOnOrOff === true) {
+    timerStart();
+  }
+}
+
+setInterval(timerGoOrNo, 10);
+
+function timerSet() {
+  timerOnOrOff = false;
+  let timeInput = prompt("Please enter the time you would like to set in this format (Mins:Seconds:Milliseconds)", "00:00:00");
```

Comment:
TimeInput is already declared above (let timeInput). Either delete that declaration or don't redeclare it here, just define it (timeInput = prompt(""))